src/ukify/test/test_ukify.py

```diff
@@ -891,5 +891,70 @@ def test_key_cert_generation(tmp_path):
     assert 'Certificate' in out
     assert re.search(r'Issuer: CN\s?=\s?SecureBoot signing key on host', out)
 
+@pytest.mark.skipif(not slow_tests, reason='slow')
+def test_join_pcrsig(capsys, kernel_initrd, tmp_path):
+    if kernel_initrd is None:
+        pytest.skip('linux+initrd not found')
+    try:
+        systemd_measure()
+    except ValueError:
+        pytest.skip('systemd-measure not found')
+
+    ourdir = pathlib.Path(__file__).parent
+    pub = unbase64(ourdir / 'example.tpm2-pcr-public.pem.base64')
+
+    output = f'{tmp_path}/basic.efi'
+    args = [
+        'build',
+        *kernel_initrd,
+        f'--output={output}',
+        f'--pcr-public-key={pub.name}',
+        '--json=short',
+        '--policy-digest',
+    ] + arg_tools
+    opts = ukify.parse_args(args)
+    try:
+        ukify.check_inputs(opts)
+    except OSError as e:
+        pytest.skip(str(e))
+
+    ukify.make_uki(opts)
+    pcrs = json.loads(capsys.readouterr().out)
+    for bank, sigs in pcrs.items():
+        for sig in sigs:
+            sig['sig'] = 'a' * int(bank[3:])
+
+    opts = ukify.parse_args(['inspect', output])
+    ukify.inspect_sections(opts)
+    text = capsys.readouterr().out
+    assert re.search(fr'\.pcrpkey', text, re.MULTILINE)
+    assert re.search(fr'\.pcrsig', text, re.MULTILINE)
+    assert not re.search(fr'"sig":', text, re.MULTILINE)
```

Comment:
None of this is interpolating anything
```suggestion
    assert re.search(r'\.pcrpkey', text, re.MULTILINE)
    assert re.search(r'\.pcrsig', text, re.MULTILINE)
    assert not re.search(r'"sig":', text, re.MULTILINE)
```